shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rdl/RDLBackendHandlerFactory.java

```diff
@@ -91,6 +93,9 @@ private static void checkRegistryCenterExisted(final SQLStatement sqlStatement)
         if (sqlStatement instanceof CreateShardingRuleStatement) {
```

Comment:
This branch will be removed later, won't it?

Reply:
Yes, next PR.